crypto/fipsmodule/kem/internal.h

```diff
@@ -98,6 +98,15 @@ int KEM_KEY_set_raw_secret_key(KEM_KEY *key, const uint8_t *in);
 int KEM_KEY_set_raw_key(KEM_KEY *key, const uint8_t *in_public,
                                       const uint8_t *in_secret);
 
+// KEM_KEY_set_raw_keypair_from_seed function generates a keypair from the
+// given seed using the appropriate key generation function based on the
+// KEM variant, then allocates and sets both public and secret key buffers
+// within the given |key|.
```

Comment:
NP: Should we note the state of `KEM_KEY` is expected to be in? That is, `key->kem` is expected to be initialized, but `key->public_key` and `key->secret_key` should be NULL.